### DISCUSSION TITLE
Can I set up a Shields.io to display GitHub pages deployment status?

### DISCUSSION THREAD

**Discussion by DylanDevelops** [badges/shields#8817] [Q&A] [answered]
there is a built-in option in GitHub for this but it doesn't have the formatting I would like, I'm trying to create a shield.io badge for it but I'm not sure how to here.

<img width="829" alt="Screenshot 2023-01-17 at 11 39 51 AM" src="https://user-images.githubusercontent.com/48571264/212983569-933036bb-bc8f-4189-88f9-0957112792b7.png">

Does anyone know what I would need to add to this to make it work with my website's deployment?

**Comment by chris48s** [comment, marked as answer]
https://github.com/actions/dylanravel.com is not a repo - you'd need to use the repo your GH pages site is deployed from and then the filename of the workflow that deploys it.

For example, in our repo https://github.com/badges/shields there is a GH pages site ( https://contributing.shields.io/ ) deployed by the workflow file https://github.com/badges/shields/blob/master/.github/workflows/deploy-docs.yml

so the corresponding badge for that workflow is https://img.shields.io/github/actions/workflow/status/badges/shields/deploy-docs.yml

**Reply by msr8** [reply]
Do you have any idea what to do incase the deoplyment was automatic? as in no workflow file was set, for eg in this repo: https://github.com/msr8/tictactoe

**Reply by chris48s** [reply]
In theory https://img.shields.io/github/actions/workflow/status/msr8/tictactoe/pages/pages-build-deployment should work

In practice, I tried it, and hit a bug - but I've raised an issue to fix it at https://github.com/badges/shields/issues/9315

**Reply by chris48s** [reply]
this now works https://img.shields.io/github/actions/workflow/status/msr8/tictactoe/pages/pages-build-deployment
